cmd/skaffold/app/cmd/config/flags.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2018 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var configFile, kubectx string
+
+func AddConfigFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVarP(&configFile, "config", "c", "", "path to skaffold config")
```

Comment:
It might make more sense to have the CLI surface look like `kubectl config`

Reply:
Hmm, so you mean expose the individual values through actual subcommands for `skaffold config`? e.g. 
`skaffold config default-repo` to get the `default-repo` value, `skaffold config list` to show the whole thing?